bundle/edu.gemini.catalog/src/main/scala/edu/gemini/catalog/votable/VoTableParser.scala

```diff
@@ -244,6 +254,12 @@ case object SimbadAdapter extends CatalogAdapter {
     }
     \/-((band.flatten |@| MagnitudeSystem.fromString(p._2))((b, s) => (b, s)))
   }
+
+  def containsExceptions(xml: Node): Boolean = {
+    // The only case known is with java.lang.NullPointerException but let's make the check
+    // more general.
+    (xml \\ "INFO" \ "@value").text.matches("java\\..*Exception")
```

Comment:
Note that this was discovered by accident. Right now the backend is working fine so it is difficult to have a generic definition of what the value can be

There is no open documentation of the Simbad Votable format